Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EditorEntityModel.cpp

```diff
@@ -362,6 +362,7 @@ namespace AzToolsFramework
             if (parentInfo.HasChild(childId))
             {
                 ReparentChild(childId, entityId, AZ::EntityId());
+                AzToolsFramework::ToolsApplicationRequests::Bus::Broadcast(&ToolsApplicationRequests::RemoveDirtyEntity, childId);
```

Comment:
Aren't these changes to `EditorEntityModel` in development already?

Reply:
I don't know why but when I tried to break the original commit down this old change just got mixed into the PR.